src/main/java/uk/gov/justice/hmpps/offenderevents/OffenderEventsApplication.java

```diff
@@ -1,14 +0,0 @@
-package uk.gov.justice.hmpps.offenderevents;
-
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
-
-@SpringBootApplication
-@ConfigurationPropertiesScan
-public class OffenderEventsApplication {
-    public static void main(final String[] args) {
-        SpringApplication.run(OffenderEventsApplication.class, args);
-    }
-
-}
```

Comment:
Has this class moved?  I couldn't find it.

Reply:
I cheekily moved it to a kt file - thought I'd pushed with the previous ones

Reply:
I cheekily moved it to a kt file - thought I'd pushed with the previous ones

Reply:
Haha sorry ignore that - converted to Kotlin. :smile: